docs/development.md

```diff
@@ -92,6 +92,22 @@ Within this range, each package in the API package is given a range of 100 locat
 
 ## API Notes
 
+### API File Structure
+
+All API endpoints must reside in the corresponding file path from `internal/api`.
```

Comment:
This sentence could use some clarifying.
"corresponding" is doing too much work here.
I know you have an example below, but examples are meant to clarify not explain.